django-stubs/db/models/base.pyi

```diff
@@ -13,6 +13,7 @@ class Model(metaclass=ModelBase):
     _meta: Any
     _default_manager: Manager[Model]
     pk: Any = ...
+    objects: Manager[Model]
```

Comment:
Please, take a look at the https://github.com/mkurnikov/django-stubs/pull/20 and `_default_manager` in the Django docs. `_default_manager` is the reason why there's no `objects` attribute on the `Model` class.

Reply:
Um... then I don't understand. I should somehow define `objects` as alias to the `_default_manager`?

Reply:
Maybe I can define `objects` and `_default_manager` as a `TypeVar("_T", bound=Manager)`? Will it be appropriate?

Reply:
You probably already know this, but just in case not, you can hint objects as `objects: models.Manager` and it will work generically for everything, though it won't be as specific as something like `Manager[Model]`.  We use this in our codebase a lot.